libbeat/_meta/config.reference.yml

```diff
@@ -982,6 +982,9 @@ logging.files:
   # never, once, and freely. Default is never.
   #ssl.renegotiation: never
 
+  #stats.period: 10s
+  #state.period: 1m
```

Comment:
`stats/state`? I wonder how many users will get confused by this (or how many typos we will see). How about `metrics.period` and `state.period`?